qiskit/circuit/instruction.py

```diff
@@ -269,12 +269,17 @@ def soft_compare(self, other: "Instruction") -> bool:
         return True
 
     def _define(self):
-        """Populates self.definition with a decomposition of this gate."""
+        """Populate the cached :attr:`_definition` field of this :class:`Instruction`.
```

Comment:
is there any advantage to refer with `:attr:` if it has no docstring?

Reply:
Not really, but I don't think it particularly hurts either.  At one point I think I was considering documenting `_definition` as well - I can do that if you'd rather?